internal/cluster/distribution/eks/eksworkflow/workflow_update_node_pool.go

```diff
@@ -42,20 +54,22 @@ type UpdateNodePoolWorkflowInput struct {
 	NodeImage string
 }
 
-func UpdateNodePoolWorkflow(ctx workflow.Context, input UpdateNodePoolWorkflowInput) (err error) {
+func (w UpdateNodePoolWorkflow) Register() {
+	workflow.RegisterWithOptions(w.Execute, workflow.RegisterOptions{Name: UpdateNodePoolWorkflowName})
+}
+
+func (w UpdateNodePoolWorkflow) Execute(ctx workflow.Context, input UpdateNodePoolWorkflowInput) (err error) {
 	activityOptions := workflow.ActivityOptions{
 		ScheduleToStartTimeout: time.Duration(workflow.GetInfo(ctx).ExecutionStartToCloseTimeoutSeconds) * time.Second,
 	}
 
 	ctx = workflow.WithActivityOptions(ctx, activityOptions)
 
-	proc := process.Start(
-		workflow.WithStartToCloseTimeout(ctx, 10*time.Minute),
-		input.OrganizationID,
-		fmt.Sprint(input.ClusterID),
-	)
+	clusterID := brn.New(input.OrganizationID, brn.ClusterResourceType, fmt.Sprint(input.ClusterID))
```

Comment:
Two ideas here:

- provide a BRN to the workflow in the first place?
- use UID instead of the numeric ID?

Reply:
it is not strictly numeric (that is why I choose string for the resource id)

this should be something that is user friendly on the other end